src/wp-includes/functions.php

```diff
@@ -6026,16 +6023,15 @@ function _doing_it_wrong( $function_name, $message, $version ) {
 				'https://wordpress.org/documentation/article/debugging-in-wordpress/'
 			);
 
-			trigger_error(
-				sprintf(
-					'Function %1$s was called <strong>incorrectly</strong>. %2$s %3$s',
-					$function_name,
-					$message,
-					$version
-				),
-				E_USER_NOTICE
+			$message = sprintf(
+				'Function %1$s was called <strong>incorrectly</strong>. %2$s %3$s',
+				$function_name,
+				$message,
+				$version
 			);
 		}
+
+		wp_trigger_error( $function_name, $message );
```

Comment:
Whoopsie, should not be passing the function name. `_doing_it_wrong()` already added it to the message.

Fixed in https://github.com/WordPress/wordpress-develop/pull/5161/commits/0293e8f60bc6d0ce1d28de7a7169d85bca54daba ✅